devops/dockerfiles/origin-auth-server

```diff
@@ -0,0 +1,23 @@
+FROM node:10
+
+WORKDIR /app
+
+ENV NODE_ENV production
+
+# Install envkey-source to make environment available for sequelize migration
+RUN curl -s -L -o envkey-source.tar.gz https://github.com/envkey/envkey-source/releases/download/v1.2.5/envkey-source_1.2.5_linux_amd64.tar.gz
+RUN tar -zxf envkey-source.tar.gz 2> /dev/null
+RUN rm envkey-source.tar.gz
+RUN mv envkey-source /usr/local/bin
+
+COPY package.json ./
+COPY yarn.lock ./
+COPY lerna.json ./
+COPY ./infra/auth-utils ./infra/auth-utils
```

Comment:
Have you considered putting this under packages/auth-utils rather than in infra ? I think overall we are trying to use infra for servers and packages for any common library. Not a huge deal though and this should not be addressed in this PR but rather in #3807

Reply:
Actually, it's under packages. auth-server is the one under infra. A mistake from my side here.